core/runtime/v2/manager.go

```diff
@@ -568,3 +575,83 @@ func (m *TaskManager) PluginInfo(ctx context.Context, request interface{}) (inte
 	}
 	return &info, nil
 }
+
+func (m *TaskManager) validateRuntimeFeatures(ctx context.Context, opts runtime.CreateOpts) error {
+	var spec specs.Spec
+	if err := typeurl.UnmarshalTo(opts.Spec, &spec); err != nil {
+		return fmt.Errorf("unmarshal spec: %w", err)
+	}
+
+	// Only ask for the PluginInfo if idmap mounts are used.
+	if !usesIDMapMounts(spec) {
+		return nil
+	}
+
+	pInfo, err := m.PluginInfo(ctx, &apitypes.RuntimeRequest{RuntimePath: opts.Runtime})
```

Comment:
PluginInfo takes a PluginInfo request.
See https://github.com/containerd/containerd/pull/9713

Reply:
I don't follow, I'm calling `m.PluginInfo()` that is: https://github.com/containerd/containerd/blob/805ed8e871537b8105aa94fcde04e35e3940f2ff/core/runtime/v2/manager.go#L540-L541

It tests if the type is: `apitypes.RuntimeRequest`.

What am I missing?

Reply:
Sorry, I thought this was another method in a different class